strawberryfields/backends/base.py

```diff
@@ -263,17 +264,19 @@ def prepare_squeezed_state(self, r, phi, mode):
         """
         raise NotImplementedError
 
-    def prepare_displaced_squeezed_state(self, alpha, r, phi, mode):
+    def prepare_displaced_squeezed_state(self, r_d, phi_d, r_s, phi_s, mode):
```

Comment:
Could it make sense to accept `(r, phi)` tuples to each function and unpacking it within? This would perhaps just help out with grouping the arguments as the number of arguments grows for each signature (though might not be such an easy change, so let me know!).

Reply:
Yes of course, that's a possibility too. I'll go with what the majority prefers!

Reply:
Had a chat on this with Josh and Nico and they were in favour of having it as is (without having an explicit tuple), so I'm also on board with the original solution! Just wanted to update you :slightly_smiling_face:

Reply:
Thank you Antal. On my end, I'm just going through all the broken tests, there are a few that I don't understand and they need a bit of investigating...